.gitignore

```diff
@@ -0,0 +1,4 @@
+target/
+/build/opencast-influxdb-adapter-1.0.jar
```

Comment:
Wouldn't ignoring `build` make more sense, rather than the jar + version?

Reply:
Oh, absolutely, I just missed that. Will be fixed soon.